admin/installer/_images.py

```diff
@@ -53,7 +52,7 @@ class AWS_REGIONS(Values):
 
 DEFAULT_IMAGE_BUCKET = u'clusterhq-installer-images'
 DEFAULT_BUILD_REGION = AWS_REGIONS.US_WEST_1
-DEFAULT_DISTRIBUTION = u"ubuntu-14.04"
+DEFAULT_DISTRIBUTION = u"ubuntu-16.04"
```

Comment:
There was a plan to have cloudformation templates for other distributions, but we never got round to it....and probably never will.
I'll remove this.